bin/deploy_to_balena.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+set -euo pipefail
+
+while [[ $# -gt 0 ]]; do
```

Comment:
Maybe we can just check environment variables instead of handling inputs via command-line args.